docs/dev/baremetal.md

```diff
@@ -35,6 +35,31 @@ Forking is painful, but ...
   openshift/installer, it's quite beneficial for us to be working in
   that codebase from the start.
 
+## What's next?
+
+### Bootstrap Ignition Customizations
+
+We need to make bare metal specific bootstrap ignition customizations.
+How best to do it? With per platform directories for assets?
```

Comment:
@celebdor was going to look into this - I'm not sure static files are enough in all cases as the current dev-scripts approach does some templating.  It'd be nice to avoid doing much stuff in the golang assets though, it's kinda ugly writing go code to generate scripts etc, although there is some of that in openshift-install already..